runtime/src/accounts_db.rs

```diff
@@ -999,6 +999,9 @@ pub struct AccountsDb {
     /// Keeps tracks of index into AppendVec on a per slot basis
     pub accounts_index: AccountInfoAccountsIndex,
 
+    /// slot that is one epoch older than the highest slot where accounts hash calculation has completed
+    pub accounts_hash_complete_one_epoch_old: RwLock<Slot>,
```

Comment:
Could this be an [`AtomicCell<Slot>`](https://docs.rs/crossbeam/latest/crossbeam/atomic/struct.AtomicCell.html) instead? Since `Slot` is a `u64`, we can avoid using a full lock if we don't actually need the mutual exclusion.

`AtomicCell`, like other atomic types, has [`fetch_update()`](https://docs.rs/crossbeam/latest/crossbeam/atomic/struct.AtomicCell.html#method.fetch_update), which could be used to handle the `max()` check.